### DISCUSSION TITLE
Email ticket response does not change status to "response received"

### DISCUSSION THREAD

**Discussion by fermic** [YetiForceCompany/YetiForceCRM#14392] [How to]
Hello guys! I need your help.

When the user sends an e-mail, the system registers the ticket and links the e-mail correctly.

However, if the ticket is finalized or changed to any other status, when the user sends a new email, the system does not reopen the ticket with the status of "response received" and also does not create a new ticket. It simply links the email without changing anything.

I already tried all the options but the behavior does not change.

Any suggestions for solving this problem? Because I need to know when the user responded to a ticket via email. 

Thank you so much for your help. 

![Screenshot_68](https://user-images.githubusercontent.com/80478401/110826141-8a6a3c00-8273-11eb-9ca7-3d3e3ca1da9a.png)

**Comment by mariuszkrzaczkowski** [comment]
how do you set the folder scan?

**Comment by fermic** [comment]
Thank you very much for the quick response

![Screenshot_1](https://user-images.githubusercontent.com/80478401/110870721-cd480600-82ab-11eb-827c-f51b0360dd1f.jpg)

**Comment by mariuszkrzaczkowski** [comment]
See the documentation https://yetiforce.com/en/knowledge-base/documentation/administrator-documentation/item/mail-scanner

![image](https://user-images.githubusercontent.com/10198654/110902570-660b6f80-8306-11eb-8725-fbaaf3867e47.png)
